packages/web/src/components/track/GiantTrackTile.module.css

```diff
@@ -294,6 +294,7 @@
   top: 0;
   /* may need design to reproduce a more pixel perfect icon to avoid the -1px below */
   left: -1px;
+  z-index: 3;
```

Comment:
I think we might have a z-index tracker somewhere, not sure

Reply:
oh yeah. adding it there too thanks